File: PathoFroh/src/main/java/com/patho/main/action/handler/AbstractHandler.java
package com.patho.main.action.handler;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import org.springframework.beans.factory.annotation.Autowired;

import com.patho.main.config.util.ResourceBundle;

import lombok.AccessLevel;
import lombok.Getter;
import lombok.Setter;

public class AbstractHandler {

	protected final Logger logger = LoggerFactory.getLogger(this.getClass());
	
	@Autowired
	@Getter(AccessLevel.NONE)
	@Setter(AccessLevel.NONE)
	protected ResourceBundle resourceBundle;
	
}

File: PathoFroh/src/main/kotlin/com/patho/main/action/handler/AbstractHandler.kt
package com.patho.main.action.handler

import com.patho.main.config.util.ResourceBundle
import org.slf4j.LoggerFactory
import org.springframework.beans.factory.annotation.Autowired

abstract class AbstractHandler {

    protected val logger = LoggerFactory.getLogger(this.javaClass)

    @Autowired
    protected lateinit var resourceBundle: ResourceBundle

    /**
     * Initializes the current view
     */
    abstract fun loadHandler()
}